Assets/Scripts/Player/TopDownMovement.cs

```diff
@@ -37,4 +39,9 @@ void Update()
             hitBox.velocity = moveInput * moveSpeed;
         }
     }
+
+    public void OnMove(InputAction.CallbackContext context)
+    {
+        moveInput = context.ReadValue<Vector2>();
+    }
```

Comment:
I'm not too familiar with the new input system. is this vector normalized?

Reply:
It doesn't need to be. Inputs are either 0 or 1